BitmovinConvivaAnalytics/Classes/SsaiApi.swift

```diff
@@ -0,0 +1,68 @@
+//
+// Bitmovin Player iOS SDK
+// Copyright (C) 2024, Bitmovin GmbH, All Rights Reserved
+//
+// This source code and its use and distribution, is subject to the terms
+// and conditions of the applicable license agreement.
+//
+
+import Foundation
+
+/// Enables reporting of server-side ad breaks and ads.
+public class SsaiApi {
+    internal weak var delegate: SsaiApiDelegate?
+
+    /// Checks if a server-side ad break is currently active.
+    ///
+    /// - Returns: `true` if a server-side ad break is active, `false` otherwise.
+    public var isAdBreakActive: Bool {
+        delegate?.ssaiApi_isAdBreakActive ?? false
+    }
+
+    /// Reports the start of a server-side ad break. Must be called before the first ad starts.
+    /// Has no effect if a server-side ad break is already playing.
+    public func reportAdBreakStarted() {
+        delegate?.ssaiApi_reportAdBreakStarted()
+    }
+
+    /// Reports the start of a server-side ad break. Must be called before the first ad starts.
+    /// Has no effect if a server-side ad break is already playing.
+    ///
+    /// - Parameter adBreakInfo: Dictionary containing metadata about the server-side ad break. Can be `nil`.
+    public func reportAdBreakStarted(adBreakInfo: [String: Any]?) {
+        delegate?.ssaiApi_reportAdBreakStarted(adBreakInfo: adBreakInfo)
+    }
+
+    /// Reports the end of a server-side ad break. Must be called after the last ad has finished.
+    /// Has no effect if no server-side ad break is currently playing.
+    public func reportAdBreakFinished() {
+        delegate?.ssaiApi_reportAdBreakFinished()
+    }
+
+    /// Reports the start of a server-side ad.
+    ///
+    /// - Parameter adInfo: Object containing metadata about the server-side ad.
```

Comment:
NIT: Could use some documentation that there needs to be an active ad break first.

Reply:
Good call, fixed in 2e0c820